.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - about: "Have a question about using this software, or about Solana in general? Post it on the Solana Stack Exchange."
+    name: Ask a Question
+    url: "https://solana.stackexchange.com/questions/ask"
+  - about: "Start or join a discussion on the Solana Tech Discord."
+    name: Start a Discussion
```

Comment:
I’m ok with linking to discord with more detail about the kinds of discussions we hold within it:
No dev support. Solana protocol engineering, validator operations, and Solana Labs validator client engineering